db/migrations/20220625_delete_pacticipants_with_no_name.rb

```diff
@@ -4,7 +4,7 @@
 # This migration deletes any pacticipants with null names that are not referenced by any other rows,
 # or assigns a name if the pacticipant is referenced so that it can be deleted through the API
 # (I can't think of a reason why a pacticipant with a null name should be referenced, but better to be safe than sorry).
-# The "find tables that reference pacticipants" logic is done dynamically because Pactflow has extra tables not in the OSS.
+# The "find tables that reference pacticipants" logic is done dynamically because PactFlow has extra tables not in the OSS.
```

Comment:
Can you revert this one? A change to the migration files will cause the database migrations to be run unnecessarily when we do a deployment.

Reply:
What file? 🤔 😛